src/main.ts

```diff
@@ -1,9 +1,29 @@
+import { INestApplication } from "@nestjs/common";
 import { NestFactory } from "@nestjs/core";
+import { DocumentBuilder, SwaggerModule } from "@nestjs/swagger";
+import * as dotenv from "dotenv";
 
 import { AppModule } from "./app.module";
+import { Environment } from "./config/env.validation";
 
+dotenv.config();
```

Comment:
not recommended and may not be necessary here.

Reply:
I agree.